docs/documentation/94/resultset.md

```diff
@@ -16,4 +16,14 @@ The following must be considered when using the `ResultSet` interface:
 * You must close a `ResultSet` by calling `close()` once you have finished using
 	it.
 * Once you make another query with the `Statement` used to create a `ResultSet`,
-	the currently open `ResultSet` instance is closed automatically.
\ No newline at end of file
+	the currently open `ResultSet` instance is closed automatically.
```

Comment:
This looks like it is held over from your other PR ?

Reply:
Ah, yeah.. I'm sorry, this is embarrassing :/ How do I fix this?